simpleui-server/src/sui_data.ts

```diff
@@ -112,235 +124,164 @@ export class SuiData {
         } catch (err) {
             Logger.log(LogLevel.ERROR, `Error in getPortFromRequest(): ${err}`);
         }
-
         return zmqPort;
     }
 
     static incrRequestNum() { SuiData.requestNum = (SuiData.requestNum + 1) % 100000; }
 
-    // from sui_data.php:  zmqRequest($props, $DataPortPrefix, $cmd, $valueName, $expectedResponseRoot, "get");
-    static async suiDataRequest(req: Request<ParamsDictionary>, res: Response, uiProps: any) {
-        // req contains:  /:appName/:propsStub/:tabName/query/cmd/zmq/:zmqPortExpr/:zmqCmd/:zmqValue
-        // if the req.query contains
+    /**
+     * Request data from c++ apps via zeromq. An http request can only request data from a single zmq socket
+     * @param req 
+     * @param res 
+     * @param uiProps 
+     */
+     static async zmqDataRequest(req: Request<ParamsDictionary>, res: Response, uiProps: any) {
```

Comment:
Do these need to be async? They don't appear to return a promise or block on anything...